tests/test_distributed.py

```
@@ -0,0 +1,278 @@
+
+        patched_valkey = MagicMock()
+        with patch("vector_topic_modeling.distributed.valkey", patched_valkey):
+            assert distributed.valkey is patched_valkey
+
+    importlib.reload(distributed)
+
+
+def test_valkey_import_success_sets_availability_true() -> None:
+    import importlib
+    import sys
+    import vector_topic_modeling.distributed as distributed
```

Ruff (0.15.7)

[warning] Use `from vector_topic_modeling import distributed` in lieu of alias

Replace with `from vector_topic_modeling import distributed`

(PLR0402)